addons/web/static/src/views/graph/graph_model.js

```diff
@@ -413,6 +413,10 @@ export class GraphModel extends Model {
         return processedDataPoints;
     }
 
+    _getEmptyGroupLabel(fieldName) {
+        return this.env._t("Undefined");
+    }
```

Comment:
since this PR https://github.com/odoo/odoo/pull/120094 merged we can use same function '_getDefaultFilterLabel' for our requirement and adept changes accordingly in project and helpdesk both. so no need write test cases again.